Sprite/Sprite.swift

```diff
@@ -69,7 +68,7 @@ public class Sprite: UIView {
   public func setImageNumber( n: Int ) {
     
     _imageNumber = n
-    if !sprite!.isAnimating() && seq != nil {
+    if !sprite!.isAnimating && seq != nil {
```

Comment:
If you drop the (), I don't think it'll return the function itself instead of the Bool value. Right?